src/Annotation/DocblockAnnotationParser.php

```diff
@@ -47,7 +48,12 @@ public function __construct(DocBlockFactoryInterface $factory, Formatter $tagsFo
      */
     public function parse(string $docblock): array
     {
-        $doc = $this->factory->create($docblock);
+        try {
+            $doc = $this->factory->create($docblock);
+        } catch (InvalidArgumentException $e) {
+            // happens with malformed tags
+            return [];
```

Comment:
@theofidry I'm not sure that returning an empty array here it's the best solution. Alternatively we can throw [`UnexpectedValueException`](https://www.php.net/manual/en/class.unexpectedvalueexception.php), that way it will be caught in the [PHP compactor](https://github.com/box-project/box/blob/3e0cec7a37d66fbc02f23b5d44dae0e6e9364c1f/src/Compactor/Php.php#L77-L78). WDYT?

Reply:
edit: it's not a big deal so feel free to ignore this if you don't want to spend more time on it